.golangci.yml

```diff
@@ -29,10 +30,11 @@ issues:
 linters-settings:
   errcheck:
     exclude-functions:
+      # Don't flag lines such as "io.Copy(io.Discard, resp.Body)".
+      - io.Copy
+      - io.WriteString
```

Comment:
I see we use io.Copy at https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/operator/gzip_config.go#L44, may we would find alternative instead of excluding.

I couldn't find `io.WriteString` though

Reply:
Thanks @slashpai 
I Got it. Remove it frist.